src/connect.rs

```diff
@@ -76,6 +76,13 @@ macro_rules! plumb_bool {
         }
     };
 }
+macro_rules! plumb_bool_2arg {
```

Comment:
nice!

Reply:
not super elegant, but this bool option takes an argument where all the others do not